src/components/views/rooms/Stickerpicker.tsx

```diff
@@ -123,7 +123,7 @@ export default class Stickerpicker extends React.PureComponent<IProps, IState> {
             logger.warn('No widget ID specified, not disabling assets');
         }
 
-        this.props.setShowStickers(false);
+        this.props.setStickersVisible(false);
```

Comment:
in the same vein maybe `setStickerPickerOpen(...)`

Reply:
Done in 0e9fd3e49f